Makefile

```diff
@@ -1,25 +1,25 @@
 create-servers:
 	packer build packer/compute.packer.json
 
-init:
-	./create.sh $(cluster)
+create:
+	./create.sh $(workspace) $(workspaces)
```

Comment:
```suggestion
	./create.sh $(workspace) $(path)
```

Reply:
That makes more sense. I originally had WORKSPACES_PATH, but it was awkwardly long. No idea why I didn't just go with path